### PR TITLE
Suricata dashboard refresh

### PR DESCRIPTION
**Pull request by peasead**
Redid an export after an index refresh.

### CODE REVIEW

**Comment by neu5ron** [comment]
best way to test is to do a manual import (via kibana) or automatic import (using rock dashboards kibana import script).

It's hard to review export updates because it updates every single file (because of timestamp or kibana version changes). So if you do that and it works, then LGTM.

File: ecs-configuration/kibana/config/index.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
-[
-  "7.5.1"
-]
+-n [
```
Comment (dcode) [inline ecs-configuration/kibana/config/index.json:1]: Whats going on with these `-n`? This doesn't look right

**Comment by peasead** [comment]
Agreed, it does look weird.

I fixed them, verified that they worked, ran the export.

Open to more testing suggestions.

**Comment by peasead** [comment]
Steps to test this:
1. built a working ROCK VM
1. made the dashboard changes to update Suricata to the new Logstash changes made for the SIEM
1. refreshed the Index Pattern for `ecs-*` and `ecs-suricata-*`
1. ran a pcap through to ensure all dashboards were populated
1. ran the export saved objects script from this repo
1. pushed the exported objects into this branch
1. downloaded this branch as a tarball
1. renamed the contents of the tarball from `suricata-dashboard-refresh` to `rock-dashboards-master`
1. retarred the `rock-dashboards-master` folder as `rock-dashboards_master.tar.gz`
1. copied this tarball to `/srv/rocknsm/support/` of a fresh ROCK vm
1. ran the `rock setup` process
1. verified that the dashboards loaded
1. verified that pcap populated the dashboards

**Pull request merged**